### PR TITLE
AUTO-155: create config for iPhone 6, iPhone SE (2nd generation) and iPhone 11 Pro

### PR DESCRIPTION
**Pull request by jh-bate**
- smoke tests (not all tests) now setup so we can run against different device sizes
- build updated to match ci build
- ability to now load mock therapy settings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,12 @@ All tests
 
 `npm run test-all`
 
+## Device Configurations
+
+- iPhone 11 Pro `ios.sim.debug-11pro`
+- iPhone 6 `ios.sim.debug-6`
```
Comment (darinkrauss) [inline README.md:20]: [nit] Consider more explicit naming (as "6" could be anything). For example "ios.sim.debug.iphone-11-pro", "ios.sim.debug.iphone-6", "ios.sim.debug.iphone-se-2".

**Review by darinkrauss — APPROVED** [review]
Just one nit for your consideration, but otherwise LGTM! 🚓

File: README.md (hunk 1 above)
Comment (jh-bate) [reply README.md:20]: I was thinking `ios` implied `iphone` hence i didn't add that. Thoughts??

Comment (darinkrauss) [reply README.md:20]: Ah, I suppose. I was interpreting that was "ios.sim" - iOS Simulator, "debug" - Debug build, and everything else was the device. Up to you, but "6" could be interpreted as a version or a device type.

Comment (jh-bate) [reply README.md:20]: makes sense - have updated in the next PR. Thanks

**Pull request merged**